src/testWithSpringSecurity_5_7/java/org/openrewrite/spring/security5/UpgradeSpringSecurity61Test.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.spring.security5;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.Recipe;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.marker.RecipesThatMadeChanges;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+import org.openrewrite.test.TypeValidation;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.openrewrite.java.Assertions.java;
+
+class UpgradeSpringSecurity61Test implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipeFromResources("org.openrewrite.java.spring.security6.UpgradeSpringSecurity_6_1")
```

Comment:
Can we narrow this down to one particular recipe that's making the incorrect change? Or a combination of two that does not run through the whole migration? Ideally if we find just one we move the tests to the respective test class.